tests/integration/__init__.py

```diff
@@ -548,7 +548,7 @@ def use_cluster(cluster_name, nodes, ipformat=None, start=True, workloads=None,
                     CCM_CLUSTER = CCMCluster(path, cluster_name, **ccm_options)
                 CCM_CLUSTER.set_configuration_options({'start_native_transport': True})
                 if IS_SCYLLA:
-                    CCM_CLUSTER.set_configuration_options({'experimental': True})
+                    CCM_CLUSTER.set_configuration_options({'experimental_features': ['lwt', 'udf']})
```

Comment:
I'm not sure I understand how the patch description is related to what the patch actually does.

Reply:
experimental: True, enable all experimental features.

CDC is causing an issue (can't start cluster with multiple seeds)

Selecting only the once we need for tests, i.e. anything but CDC.

Reply:
@fruch Got it.
Please, add this explanation as a comment above the corresponding code line.